impl/src/main/java/com/sevenbridges/apiclient/impl/task/DefaultTaskCriteria.java

```diff
@@ -24,14 +25,21 @@
 import com.sevenbridges.apiclient.task.TaskOptions;
 import com.sevenbridges.apiclient.task.TaskStatus;
 
+import java.text.DateFormat;
+import java.util.Date;
+
 public class DefaultTaskCriteria extends DefaultCriteria<TaskCriteria, TaskOptions> implements TaskCriteria {
 
+  private final DateFormat dateFormat;
```

Comment:
maven findbugs plugin complains about `STCAL_INVOKE_ON_STATIC_DATE_FORMAT_INSTANCE` http://findbugs.sourceforge.net/bugDescriptions.html

This is the reason why I transferred it to an instance level. Do you have other suggestion?